etc/cvmfs/domain.d/cern.ch.conf

```diff
@@ -1,7 +1,7 @@
 CVMFS_FALLBACK_PROXY=
 . ../common.conf
 
-if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ] || [ "$CVMFS_HTTP_PROXY" = "auto;DIRECT" ]; then
+if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ]; then
```

Comment:
This change would eliminate the use of Cloudflare whenever WPAD is used, including for the CernVM default and LHC@Home.  Why would you want to do that?  Same question for EGI, OSG, and HSF.

Reply:
Eventually I came to the conclusion that for cvmfs-config-default I don't want to judge based on the proxy settings whether or not to use the OpenHTC servers.  This setting is now explicit (`CVMFS_ENABLE_OPENHTC`) and should be set for the outreach & volunteer use cases.  Our professional users should connect to our normal services.

Reply:
I strongly disagree but you also feel strongly about it and I wasn't able to persuade you so I'm letting it go.